scripts/protopackage.sh

```diff
@@ -16,7 +16,7 @@ if [[ ! -z "$3" ]]; then
 fi
 
 
-if ! grep -q 'option\s\+go_package\s\+=\s\+.*;' $FNAME; then 
+if ! grep -q 'option\s\+go_package\s\+=\s\+.*;' $FNAME; then
```

Comment:
We should be able to just delete this file and use the `go_package` variable directly both here and in #8065.

Reply:
I think I'll disentangle/remove this in a follow-up PR.